app/assets/stylesheets/donor.css.scss

```diff
@@ -0,0 +1,230 @@
+@mixin clear-list-margins() {
+  margin: 0;
+  list-style-type: none;
+}
+
+@mixin card-header-arrow() {
+  display: inline-block;
+
+  .fa {
+    padding-top: 12px;
+  }
+
+  a {
+    color: $dark-grey;
+    &:hover {
+      color: $white;
+    }
+  }
+}
+
+
+.profile-summary-donate-link {
+  background: $white;
+  border: 1px solid $donor-profile-border-grey;
+  border-radius: 4px;
+  color: $medium-green;
+  margin: 0 auto;
+  padding: 5px 10px;
+}
+
+.update-profile-link {
+  background: $donor-profile-light-grey;
+  color: $donor-profile-darker-grey;
+  padding: 8px;
+  text-decoration: none;
+  width: 100px;
+}
+
+.donor-dashboard-block {
+  @include full-width-block($off-white);
+  @include block-padding();
+  min-height: 900px;
+}
+
+.dashboard {
+  border: 1px solid $dark-grey;
+  @include drop-shadow(1);
+}
+
+.card-header {
+  @extend .text-center;
+  border-bottom: 1px solid $dark-grey;
+  background-color: $medium-green;
+
+  .card-header-title {
+    display: inline-block;
+    font-size: rem-calc(18);
+    font-weight: 400;
+    color: $white;
+    margin-top: 8px;
+    margin-bottom: 8px;
+  }
+
+  .card-header-arrow-left {
+    @include card-header-arrow();
+    margin-right: 30px;
+  }
+
+  .card-header-arrow-right {
+    @include card-header-arrow();
+    margin-left: 30px;
+  }
+}
+
+.card-profile {
+  padding-top: 30px;
+  background-color: $white;
+  border-bottom: 1px solid $dark-grey;
+  color: $charcoal;
+}
+
+.profile-info {
+  .fa-li {
+    padding-top: 2px;
+  }
+
+  .inline-list-food {
+    @extend .inline-list;
+    padding-top: 0px;
+  }
+
+  .additional-info-list {
+    margin-top: -20px;
+  }
+}
+
+.profile-img-wrap {
+  @include clear-list-margins();
+
+  .profile-img-item {
+    text-align: center;
+  }
+
+  .profile-img {
+    max-height: rem-calc(200px);
+    padding: rem-calc(15px);
+  }
+}
+
+.profile-summary-wrap {
+  @include clear-list-margins();
+
+  .profile-summary-item {
+    padding-left: rem-calc(50px);
+    padding-right: rem-calc(50px);
+
+    .profile-summary-count {
+      float: right;
+    }
+  }
+}
+
+.profile-info-title {
+  @extend .text-center;
+  color: $charcoal;
+  padding-bottom: rem-calc(15px);
+  font-size: rem-calc(24);
+}
+
+.profile-summary-title {
+  @extend .text-center;
+  color: $charcoal;
+  font-size: rem-calc(18);
+}
+
+.card-history {
+  background-color: $white;
+
+  .history-title {
+    @extend .text-center;
+    color: $white;
```

Comment:
Properties should be ordered background-color, border-bottom, color, font-size, font-weight, padding-bottom, padding-top